data/test-cases/plugins/ms-teams-sync/ui-integration-phase-3/Error_in_linking.md

```diff
@@ -0,0 +1,60 @@
+---
+# (Required) Ensure all values are filled up
+name: "Error while linking a channel on MM"
+status: Active
+priority: Normal
+folder: ui-integration-phase-3
+authors: "@arush-vashishtha"
+team_ownership: []
+priority_p1_to_p4: P2 - Core Functions (Do core functions work?)
+
+# (Optional)
+location: null
+component: null
+tags: []
+labels: []
+tested_by_contributor: null
+
+# (Optional) Test type and tools
+cypress: null
+detox: null
+mmctl: null
+playwright: null
+rainforest: []
+manual_test_environments: []
+
+# Do not change
+id: null
+key: null
+created_on: null
+last_updated: null
+case_hashed: null
+steps_hashed: null
+---
+
+**Step 1**
+
+1. Any error occured while linking a channel from RHS of MS Teams sync plugin on MM.
+
+**Step 2**
+
+1. Any error occured while linking a channel from RHS of MS Teams sync plugin on MM.
+2. Click on `Try Again` button in the `Unable to link channels` modal.
+
+**Step 3**
+
+1. Any error occured while linking a channel from RHS of MS Teams sync plugin on MM.
+2. Click on `Cancel` or `x` button in the `Unable to link channels` modal.
+
+**Step 4**
+
+1. Connect the channel on MM to channel on MS Teams using slash command `/msteams-sync connect`.
+2. Click on the MS Teams sync icon in App bar or channel header and open the RHS.
```

Comment:
Instead of click, we can add open the RHS through channel header or app bar. Fix everywhere.

Reply:
in the early test cases we have mentioned the click step, so i think it's better to keep it that way.